src/pages/en/guides/markdown-content.md

```diff
@@ -324,7 +352,6 @@ export default {
     rehypePlugins: [
       // Add a Rehype plugin that you want to enable for your project.
       // If you need to provide options for the plugin, you can use an array and put the options as the second item.
-      // 'rehype-slug',
       // ['rehype-autolink-headings', { behavior: 'prepend'}],
     ],
   },
```

Comment:
If we indent this code block (3 or 4 spaces works I think), it will nest under number 2 in the list rather than appearing as a block after the end of the `<ol>`. Same goes presumably for the text and code block that follow.

Reply:
Done! Take a look

Reply:
Thanks! This is a bit of a nitpick: I would probably also indent the code block that follows, but I could understand if you don’t think that belongs under item **2.**

<img width="816" alt="image" src="https://user-images.githubusercontent.com/357379/166049286-ca93902b-fc3a-44d8-9da5-aa8c66d52fff.png">

Reply:
Yes, it's nicer indented :smile: